deltachat-ios/Chat/InputBarAccessoryView/InputTextView.swift

```diff
@@ -76,13 +76,15 @@ open class InputTextView: UITextView {
         label.text = "Aa"
         label.backgroundColor = .clear
         label.translatesAutoresizingMaskIntoConstraints = false
+        label.isAccessibilityElement = false
         return label
     }()
     
     /// The placeholder text that appears when there is no text
     open var placeholder: String? = "Aa" {
```

Comment:
Please use the localized "write_message_desktop" string.

Reply:
fixed with https://github.com/deltachat/deltachat-ios/pull/1558/commits/ead96cf010a4d3eb3726c4a1170fae938c1a6344